src/lib/response-triggers/location.ts

```diff
@@ -0,0 +1,105 @@
+import Trigger from '../../models/trigger';
+import { DbLocation } from '../database/db-interfaces';
+import Config from '../config-interface';
+import { MessageObject } from 'webex/env';
+import { Role } from '../../models/database';
+import { Command } from '../../models/command';
+import constants from '../../constants';
+import xmlMessageParser from '../parsers/xmlMessageParser';
+import { LocationAction } from '../../models/location-action';
+import __logger from '../logger';
+
+export default class Location extends Trigger {
```

Comment:
maybe LocationConfig? Does this set locations for users or in general or both?

Reply:
Yeah I agree, LocationConfig would be clearer. Location sounds like it represents the location itself.

Reply:
Done